tls/client-tls-bio.c

```diff
@@ -79,7 +79,11 @@ int main(int argc, char** argv)
     }
 
     /* Create and initialize WOLFSSL_CTX */
+#ifdef USE_TLSV13
+    if ((ctx = wolfSSL_CTX_new(wolfTLSv1_3_client_method())) == NULL) {
```

Comment:
Don't have '{' in ifdef.
And in other examples too.

Reply:
Done